2007-01-30  Geoffrey Keating  <geoffk@apple.com>

	* config/rs6000/darwin.h (MD_UNWIND_SUPPORT): Don't define for
	64-bit.

Index: gcc/config/rs6000/darwin.h
===================================================================
--- gcc/config/rs6000/darwin.h	(revision 121440)
+++ gcc/config/rs6000/darwin.h	(working copy)
@@ -449,7 +449,9 @@
 #include <stdbool.h>
 #endif
 
+#ifndef __LP64__
 #define MD_UNWIND_SUPPORT "config/rs6000/darwin-unwind.h"
+#endif
 
 #define HAS_MD_FALLBACK_FRAME_STATE_FOR 1
 
